packages/wallet/src/redux/__tests__/initialized.test.ts

```diff
@@ -4,7 +4,7 @@ import * as states from './../state';
 import * as actions from './../actions';
 import * as scenarios from './test-scenarios';
 import { PlayerIndex, WalletProtocol } from '../types';
-import * as Funding from '../protocols/funding/reducer';
+import * as fundProtocol from '../protocols/funding';
```

Comment:
```suggestion
import * as fundingProtocol from '../protocols/funding';
```

Reply:
I was toying around with a different naming convention: `fundProtocol`, `challengeProtocol`, etc. I take it you prefer not to adopt that convention?

Reply:
That seems fine!